src/desktop/apps/artsy_primer/personalize/test/client/artists.test.coffee

```diff
@@ -117,7 +117,7 @@ describe 'ArtistsView', ->
 
   describe '#render', ->
     describe 'collector_level 2', ->
-      it 'renders the template with the correct copy', ->
+      xit 'renders the template with the correct copy', ->
```

Comment:
Out of curiosity, why would these tests break after this change? Was it an issue where the file wasn't running at all before?

Reply:
Yeah all the ones that are broken were not actually being run before -- currently assembling some #'s on how many were not run to confirm.

Reply:
cool cool -- that's been a problem in the past; whole files being skipped.